src/Altinn.App.Api/Controllers/PdfController.cs

```diff
@@ -0,0 +1,122 @@
+        [HttpGet("format")]
+        public async Task<ActionResult> GetPdfFormat(
+            [FromRoute] string org,
+            [FromRoute] string app,
+            [FromRoute] int instanceOwnerPartyId,
+            [FromRoute] Guid instanceGuid,
+            [FromRoute] Guid dataGuid)
+        {
+            Instance instance = await _instanceClient.GetInstance(app, org, instanceOwnerPartyId, instanceGuid);
+            if (instance == null)
+            {
+                return NotFound("Did not find instance");
+            }
+
+            string taskId = instance.Process?.CurrentTask?.ElementId;
+            if (taskId == null)
+            {
+                return Conflict("Instance does not have a valid currentTask");
+            }
+
+            DataElement dataElement = instance.Data.FirstOrDefault(d => d.Id == dataGuid.ToString());
+            if (dataElement == null)
+            {
+                return NotFound("Did not find data element");
+            }
+
+            string appModelclassRef = _resources.GetClassRefForLogicDataType(dataElement.DataType);
+            Type dataType = _appModel.GetModelType(appModelclassRef);
+
+            string layoutSetsString = _resources.GetLayoutSets();
+            LayoutSets layoutSets = null;
+            LayoutSet layoutSet = null;
+            if (!string.IsNullOrEmpty(layoutSetsString))
+            {
+                layoutSets = JsonSerializer.Deserialize<LayoutSets>(layoutSetsString)!;
+                layoutSet = layoutSets.Sets?.FirstOrDefault(t => t.DataType.Equals(dataElement.DataType) && t.Tasks.Contains(taskId));
+            }
+
+            string layoutSettingsFileContent = layoutSet == null ? _resources.GetLayoutSettingsString() : _resources.GetLayoutSettingsStringForSet(layoutSet.Id);
+
+            LayoutSettings layoutSettings = null;
+            if (!string.IsNullOrEmpty(layoutSettingsFileContent))
+            {
+                layoutSettings = JsonSerializer.Deserialize<LayoutSettings>(layoutSettingsFileContent);
+            }
+
+            // Ensure layoutsettings are initialized in FormatPdf
+            layoutSettings ??= new();
+            layoutSettings.Pages ??= new();
+            layoutSettings.Pages.ExcludeFromPdf ??= new();
+            layoutSettings.Pages.Order ??= new();
+            layoutSettings.Components ??= new();
+            layoutSettings.Components.ExcludeFromPdf ??= new();
+
+            object data = await _dataClient.GetFormData(instanceGuid, dataType, org, app, instanceOwnerPartyId, new Guid(dataElement.Id));
+
+            layoutSettings = await _pdfFormatter.FormatPdf(layoutSettings, data);
+
+            var result = new
+            {
+                ExcludedPages = layoutSettings.Pages.ExcludeFromPdf,
+                ExcludedComponents = layoutSettings.Components.ExcludeFromPdf,
+                PageOrder = layoutSettings.Pages.Order,
+            };
+            return Ok(result);
+        }
+    }
+}
```

Comment:
Add trailing linebreak
```suggestion
}

```